scripts/build/build_failure_rca/rca_patterns/Asset-Processing.json

```diff
@@ -7,13 +7,14 @@
         "single_line": true,
         "find_all": true,
         "log_patterns": [
-          "^.*AssetProcessor: Failed.*\.\.\.$"
+          "^.*?AssetProcessor: Failed.*\.\.\.$",
+          "^.*?AssetProcessor: Createjobs Failed:.*$"
         ],
         "log_testcases": [
-          "AssetProcessor: Failed Shaders/Terrain/TerrainDetailClipmapGenerationPass.shader"
+          "AssetProcessor: Failed Shaders/Terrain/TerrainDetailClipmapGenerationPass.shader",
+          "AssetProcessor: Createjobs Failed: /data/workspace/o3de/Gems/LyShineExamples/Assets/UI/Fonts/LyShineExamples/NotoSerif/NotoSerif-Regular.xml."
```

Comment:
Is this something we should file bugs against?

Reply:
Good point! Creating a GHI is a better way to report the issue

Reply:
Do you mean filing a GHI for that specific error called out in the example or for this PR to fix the lack of reporting of CreateJobs failures?

Reply:
The specific error. Should font files in the example fail to process on a regular basis?

Reply:
No its pretty rare to see that, I think the example I pulled from was due to some ScriptCanvas shutdown issue.  I'm not sure what this block in the JSON is used for, I was just trying to follow the existing format and put an example in there.